### PR TITLE
Fixes #1690 Accepting only csv files

### LINKED ISSUE #1690: Accepting only csv files as input
**I'm submitting a ...**
- [x] bug report

**Current behaviour:**

Earlier user was able to select any type of file but user should only add csv file . 
screenshot from 2018-09-14 08-16-36

**Expected behaviour:**

Now Only CSV files are accepted. 

screenshot from 2018-09-14 08-20-23

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1690 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.

#### Changes proposed in this pull request:

-Accepting only csv files.

### CODE REVIEW

**Comment by sk9331657** [comment]
@gabru-md  Please review it.

**Comment by sk9331657** [comment]
@ParthS007 Please review it.

**Comment by sk9331657** [comment]
Ok

**Comment by sk9331657** [comment]
@gabru-md  Please review it

**Comment by sk9331657** [comment]
@ParthS007 Please review back.

File: frontend/app/styles/partials/create-badge.scss [hunk 1]
```diff
@@ -31,6 +31,3 @@
   }
 }
 
-.hide {
-  display: none !important;
```
Comment (abishekvashok) [inline frontend/app/styles/partials/create-badge.scss]: I would be happy if this file was only rolled back.

Comment (sk9331657) [reply frontend/app/styles/partials/create-badge.scss]: I have rolled back this.

**Comment by sk9331657** [comment]
@abishekvashok  @gabru-md Now it can be merged. It was my mistake . I have actually added the toggler implementation also in this PR.

File: frontend/app/templates/create-badges.hbs [hunk 2]
```diff
@@ -1,8 +1,12 @@
 <div class="ui stackable grid">
-    <div class="right floated right aligned two wide column">
-        <button {{action 'togglePreview'}} class="tiny ui basic orange button" data-tooltip="Toggle Preview" data-position="left center">
-            <i class="exchange icon"></i>
-        </button>
+    <div class="right floated right aligned two wide column" style="margin-right:50px;">
```
[the remark this answers is not in the document]
Comment (sk9331657) [reply frontend/app/templates/create-badges.hbs]: Resolved.

**Comment by sk9331657** [comment]
@ParthS007 Please review if you are free.

File: frontend/app/styles/partials/create-badge.scss (hunk 1 above)
Comment (gabru-md) [reply frontend/app/styles/partials/create-badge.scss]: why is this deleted?

**Comment by sk9331657** [comment]
@gabru-md Now it can be merged.

**Comment by ParthS007** [comment]
@sk9331657  Since you have more than 2 PR merged already. Please squash your commits into one with a good commit message from next time.

**Pull request merged**